test/integration/default/controls/services_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Temporary `if` due to `opensuse-leap-15` bug re: `service`
+if os[:name] == 'suse'
+  puts "[Skip `service`-based tests due to `opensuse-leap-15` detection bug (see https://github.com/inspec/train/issues/377)]"
```

Comment:
not needed anymore

Reply:
Ah yes, this needs to be updated in the `template-formula` as well.

Reply:
@n-rodriguez Would you mind merging https://github.com/saltstack-formulas/template-formula/pull/127?